hazelcast-jet-core/src/main/java/com/hazelcast/jet/DAG.java

```diff
@@ -229,7 +230,19 @@ public Vertex getVertex(String vertexName) {
 
     @Override
     public String toString() {
-        return "Vertices " + verticesByName + "\nEdges " + edges;
+        final StringBuilder b = new StringBuilder("dag\n");
+        for (Iterator<Vertex> it = iterator(); it.hasNext();) {
+            final Vertex v = it.next();
+            b.append("    .vertex(").append(v.getName());
+            if (v.getLocalParallelism() != -1) {
+                b.append(".localParallelism(").append(v.getLocalParallelism()).append(')');
+            }
```

Comment:
```java
.vertex("stopword-source".localParallelism(1))
```
This stings my eyes more than
```java
.vertex(stopword-source.localParallelism(1))
```
I can squint at the fact that `stopword-source` is not a legal Java identifier, but seeing a `localParallelism()` method call on a `String` looks just plain wrong.